gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfiguration.java

```diff
@@ -37,12 +37,16 @@
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.collect.ImmutableList;
 import com.google.common.flogger.GoogleLogger;
+import java.io.IOException;
 import java.util.Collection;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.fs.permission.FsPermission;
 
-/** This class provides a configuration for the {@link GoogleHadoopFileSystem} implementations. */
+/**
```

Comment:
Use Google Java Formatter to format changes: https://github.com/google/google-java-format

Reply:
I doubt we use this formatter before for these files.. the file format changes a lot after I applied google-java-format.

Reply:
You can apply it only to modified lines, not whole file with the command:
```bash
git diff -U0 origin/master | python "${HOME}/google-java-format/scripts/google-java-format-diff.py" -p1 -i
```